### PR TITLE
Adding PluginHelp.Config to label plugin

### PR DESCRIPTION
**Pull request by matthyx**
ref #10714

### CODE REVIEW

File: prow/plugins/label/label.go [hunk 1]
```diff
@@ -43,9 +43,16 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
-	// The Config field is omitted because this plugin is not configurable.
+	configString := `The label plugin will use only the existing "kind/*", "priority/*", and "area/*" labels.`
```
Comment (stevekuznetsov) [inline prow/plugins/label/label.go]: I am not sure that it's important to users which of these are config and which are built-in, so can we just have one message with a list?

```go
	labels := []string{"kind", "priority", "area"}
	labels = append(labels, "a", "b", "c")
	var formattedLabels []string
	for _, label := range labels {
		formattedLabels = append(formattedLabels, fmt.Sprintf(`"%s/*"`, label))
	}
	configString := fmt.Sprintf("The label plugin will work on %s and %s labels.", strings.Join(formattedLabels[:len(formattedLabels)-2], ", "), formattedLabels[len(formattedLabels)-1])
	fmt.Println(configString)
```

Comment (matthyx) [reply prow/plugins/label/label.go]: Cheers!

**Pull request merged**